### ISSUE TITLE
[ Enhancement ] When submitting a new post, if there are errors, display them on the screen.

### ISSUE THREAD

**Issue by dani98000** [greenpress/admin-panel#45]
Currently, when creating a new post, if you submit and get a 400 response code from the server, with attached errors, they are not displayed to the user to see what he did wrong.

**Comment by davidmeirlevy** [comment]
So currently it shows success even though the post failed to be saved?

**Comment by dani98000** [comment]
No, what I meant is that there is no feedback when a user makes a mistake and provides invalid value or no value at all to a certain field inside the form.

**Comment by davidmeirlevy** [comment]
before submitting, right?

**Comment by dani98000** [comment]
Both before and after submitting.

**Comment by benlaniado** [comment]
I tried to reproduce the bug but did get an error message.

what I did:
created a new post
then tried to create another post in the same path as the previous one and did get an error message (maybe the error should be more informative? for example the reason why it failed?)

**Comment by davidmeirlevy** [comment]
@benlaniado it should too.
